src/platform/win32_io.c

```diff
@@ -410,6 +410,14 @@ oc_io_raw_read_link_result oc_io_raw_read_link(oc_arena* arena, oc_file_desc fd)
             result.error = OC_IO_ERR_UNKNOWN;
         }
     }
+    //NOTE: convert windows backslash sep to forward slash sep
+    for(int i = 0; i < result.target.len; i++)
+    {
+        if(result.target.ptr[i] == '\\')
+        {
+            result.target.ptr[i] = '/';
+        }
+    }
```

Comment:
This is happening a ton; should there be a `oc_win_normalize_path` function or something?

Reply:
Right, I'll make a helper for that in `win32_string_helpers.c`